src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -30,7 +34,7 @@ public Line(String name, String color) {
     public Line(String name, String color, Station upStation, Station downStation, int distance) {
```

Comment:
```suggestion
    public Line(String name, String color, Section section) {
```

생성자 내부가 복잡해 보이는데요, 좀 더 간단하게 만들어 볼 수 있을것 같아요.

[1.3 생성자에 코드를 넣지 마세요.](https://github.com/seovalue/elegant_object/issues/5)

참고해주세요~

Reply:
넵 감사합니다.